extensions/amp-accordion/1.0/accordion.type.js

```diff
@@ -56,28 +57,40 @@ AccordionDef.AccordionHeaderProps;
 /**
  * @typedef {{
  *   as: (string|PreactDef.FunctionalComponent|undefined),
+ *   role: (string|undefined)
  *   className: (string|undefined),
+ *   id: (string|undefined),
  *   children: (?PreactDef.Renderable|undefined),
  * }}
  */
 AccordionDef.AccordionContentProps;
 
 /**
  * @typedef {{
- *   role: (string|undefined),
+ *   id: (string),
+ *   role: (string),
  *   onClick: (function()|undefined),
+ *   aria-controls: (string),
+ *   aria-expanded: (string),
  *   children: (?PreactDef.Renderable|undefined),
  * }}
  */
-AccordionDef.HeaderProps;
+AccordionDef.HeaderShimProps;
+// AccordionDef.HeaderShimProps is not a public API
```

Comment:
Please merge this text with the jsdoc comment that defines the type itself. E.g.

```
/**
 * This is not a public API, it exists for ........
 * @typedef {...}
 */
```

Reply:
Thanks, updated in the latest revision!